src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 
 ## Upcoming Release
+> [!IMPORTANT]
+> Preannouncement: The default interactive login experience will change from browser based to `Web Account Manager` (WAM) based on supported platforms, [learn more](https://learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/desktop-mobile/wam). Nothing changes of user operations. This will take effect from the release of **May 21st**.
```

Comment:
change to 
```
Preannouncement: The default interactive login experience will change from browser based to Web Account Manager (WAM) based on supported platforms, [learn more](
https://learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/desktop-mobile/wam).
Nothing changes of user operations. This will take effect from the release of May 21st
```
In psd1